transforms/code/license_check/python/README.md

```diff
@@ -0,0 +1,68 @@
+# License Check 
+
+Please see the set of
+[transform project conventions](../../../README.md#transform-project-conventions)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary 
+
+This filter scans the license column of an input dataset and appends the `license_status` column to the dataset.
+
+The type of the license column can be either string or list of strings. For strings, the license name is checked against the list of approved licenses. For list of strings, each license name in the list is checked against the list of approved licenses, and all must be approved.
+
+If the license is approved, the license_status column contains True; otherwise False. 
+
+## Configuration and command line Options
+
+- The optional `lc_license_column` parameter is used to specify the column name in the input dataset that contains the license information. The default column name is license.
```

Comment:
This (and others?) do not seem to match what is in the code.  Specifically lc_license_column_name.

Reply:
done